server/src/test/java/org/opensearch/extensions/RegisterTransportActionsRequestTests.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.opensearch.common.collect.Map;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.io.IOException;
+
+public class RegisterTransportActionsRequestTests extends OpenSearchTestCase {
+    private RegisterTransportActionsRequest originalRequest;
+
+    @Before
+    public void setup() {
+        this.originalRequest = new RegisterTransportActionsRequest(Map.of("testAction", Map.class));
+    }
+
+    @Test
+    public void testRegisterTransportActionsRequest() throws IOException {
+        System.out.println("HelloWorld");
+
+        BytesStreamOutput output = new BytesStreamOutput();
+        originalRequest.writeTo(output);
+        StreamInput input = output.bytes().streamInput();
```

Comment:
(Comment) I realize that `close()` is a noop here so these don't need to be closed, but some IDE linters will complain that a resource isn't closed.  Personal preference of mine. :)